src/pkg/bundle/bundle.go

```diff
@@ -52,7 +52,8 @@ func Create(b *Bundler, signature []byte) error {
 	// grab all Zarf pkgs from OCI and put blobs in OCI store
 	for i, pkg := range bundle.Packages {
 		fetchSpinner := message.NewProgressSpinner("Fetching package %s", pkg.Name)
-
+		zarfPackageName := ""
+		zarfLayerAdded := false
```

Comment:
Let's be more specific with this var name, yes it's a Zarf layer, but more specifically it's a descriptor pointing to the Zarf package's root manifest

Reply:
going with "zarfRootLayerAdded"